src/env.ts

```diff
@@ -2,17 +2,15 @@ import { useEffect, useLayoutEffect } from 'react'
 
 export const IS_SERVER =
   typeof window === 'undefined' ||
-  // @ts-ignore
+  // @ts-expect-error
   !!(typeof Deno !== 'undefined' && Deno.version && Deno.version.deno)
 
 const __requestAnimationFrame = !IS_SERVER
   ? window['requestAnimationFrame']
   : null
 
 // polyfill for requestAnimationFrame
-export const rAF = IS_SERVER
-  ? null
-  : __requestAnimationFrame
+export const rAF = __requestAnimationFrame
```

Comment:
I think we don't have to assign `null` to `rAF` for a server environment because `setTimeout` is enough and we don't use it on a server envrionment.

Reply:
We should use `IS_SERVER` if we'd like to have code that only runs on servers.